components/esp_lvgl_port/src/lvgl8/esp_lvgl_port.c

```diff
@@ -198,10 +198,10 @@ static void lvgl_port_task(void *arg)
             task_delay_ms = lv_timer_handler();
             lvgl_port_unlock();
         }
-        if ((task_delay_ms > lvgl_port_ctx.task_max_sleep_ms) || (1 == task_delay_ms)) {
+        if (task_delay_ms > lvgl_port_ctx.task_max_sleep_ms) {
             task_delay_ms = lvgl_port_ctx.task_max_sleep_ms;
-        } else if (task_delay_ms < 1) {
-            task_delay_ms = 1;
+        } else if (task_delay_ms < 5) {
```

Comment:
Why did you changed to 5?

Reply:
In the initial version of the BSP I implemented, the test results for the LVGL Music Demo were higher compared to the current version. After testing and comparison, it was found that adjusting this part to 5 leads to an improvement.